react-ui/src/components/FiveStar.jsx

```diff
@@ -1,3 +1,7 @@
+/**
+ * Display user rating with star symbols
```

Comment:
FYI, I found that this is provided by MUI: https://v4.mui.com/components/rating. So I'll probably end up replacing this (tracked in #182).